src/ReactiveDomain/ReactiveDomain.Tests/Logging/when_logging_disabled_commands_fired.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using Xunit;
+using ReactiveDomain.Bus;
+using ReactiveDomain.EventStore;
+using ReactiveDomain.Messaging;
+using ReactiveDomain.Tests.Helpers;
+using ReactiveDomain.Tests.Subscribers.QueuedSubscriber;
+using Xunit.Sdk;
+
+namespace ReactiveDomain.Tests.Logging
+{
+
+    // ReSharper disable once InconsistentNaming
+    public class when_logging_disabled_commands_fired :
+        with_message_logging_disabled,
+        IHandle<Message>
+    {
+        static when_logging_disabled_commands_fired()
+        {
+            BootStrap.Load();
+        }
+
+        private readonly Guid _correlationId = Guid.NewGuid();
+        private IListener _listener;
+
+        private readonly int _maxCountedCommands = 25;
+        private int _multiFireCount;
+        private int _testCommandCount;
+
+        private TestCommandSubscriber _cmdHandler;
+
+        protected override void When()
+        {
+            // command must have a commandHandler
+            _cmdHandler = new TestCommandSubscriber(Bus);
+
+            _multiFireCount = 0;
+            _testCommandCount = 0;
+
+            _listener = Repo.GetListener(Logging.FullStreamName);
+            _listener.EventStream.Subscribe<Message>(this);
+
+            _listener.Start(Logging.FullStreamName);
+
+            // create and fire a set of commands
+            for (int i = 0; i < _maxCountedCommands; i++)
+            {
+                // this is just an example command - choice to fire this one was random
+                var cmd = new InformUserCmd("title",
+                                        $"message{i}",
+                                        Guid.NewGuid(),
+                                        null);
+                Bus.Fire(cmd,
+                    $"exception message{i}",
+                    TimeSpan.FromSeconds(2));
+
+                Bus.Publish(
```

Comment:
Again, why publish events in a class that's supposed to be testing commands?

Reply:
ditto, typo. I cut and paste "improved" code from the test that has both commands and events.